src/core/Buttons/Button/Button.test.tsx

```diff
@@ -159,3 +159,15 @@ it('should render without label correctly', () => {
     expect(icon).toEqual(placeholderIcon);
   });
 });
+
+it('should render as anchor element', () => {
+  const { container } = render(
+    <Button as='a' href='https://example.com/'>
+      label
+    </Button>,
+  );
+
+  const button = container.querySelector('a.iui-button') as HTMLButtonElement;
+  expect(button).toHaveTextContent('label');
+  expect(button).toHaveAttribute('href', 'https://example.com/');
```

Comment:
Why different approaches? In some tests you test if `button.href equals 'link text'` and in others you have this `.toHaveAttribute`

Reply:
Ah it's because I am using the wrong type in line 170 so `button.href` is not valid. Fixed here

I don't know if there's any difference in the two approaches, but I remember noticing `toHaveAttribute` is more reliable in some cases